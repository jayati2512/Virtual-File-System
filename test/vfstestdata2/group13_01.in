createvfs DB 1024
mountvfs DB
makedir /shapes
makedir /employee
makedir /shapes circle
makedir /shapes rectangle
makedir /shapes/rectangle area
makedir /shapes rectangle perimeter
makedir /employee technician
makedir /employee/technician fresh
makedir /employee engineer
makedir /shapes/circle circumfrence
addfile /shapes read.txt /user/include
listdir /home 1 /tmp
updatefile /shapes/abc.txt update.txt 
movefile /shapes/read.txt /shapes/rectangle/area
deletedir /employee/engineer
copyfile /shapes/rectangle/read.txt /employee/engineer
exportfile /shapes/rectangle/read.txt /user/include
