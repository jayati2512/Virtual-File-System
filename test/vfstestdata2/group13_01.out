createvfs_SUCCESS
mountvfs_SUCCESS
ERR_VFS_MAKEDIR: VFS_INSUFFICIENT_ARGUMENTS
ERR_VFS_MAKEDIR: VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
ERR_VFS_MAKEDIR: DIRECTORY_ALREADY_EXISTS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
ERR_VFS_ADDFILE: UNABLE_TO_OPEN_SPECIFIED_DATAFILE_PATH
listdir_FAILURE "CANNOT_FIND_SPECIFIED_PATH_OR_DIR"
updatefile_FAILURE "SOURCE_FILE_NOT_FOUND"
ERR_VFS_MOVEFILE: CANNOT_FIND_SPECIFIED_SOURCEFILE
deletedir_SUCCESS
ERR_VFS_COPYFILE: CANNOT_FIND_SPECIFIED_SOURCEFILE
ERR_VFS_EXPORTFILE: CANNOT_FIND_SPECIFIED_SOURCEFILE
